src/containers/HeatMap/heatMap.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Icon } from 'antd';
+import ReactTooltip from 'react-tooltip';
+import CalenderHeatMap from 'react-calendar-heatmap';
+import moment from 'moment';
+
+import LogoHeader from '../../components/LogoHeader';
+import JournalCard from '../../components/JournalCard';
+import NavigationBar from '../../components/navigationBar';
+import { ReactComponent as NoJournals } from '../assets/images/noJournals.svg';
+import toolTipData from '../../utils/toolTipData';
+import { firstDay, lastDay } from '../../utils/dates';
+import 'react-calendar-heatmap/dist/styles.css';
+import './heatMap.css';
+
+const heatMap = props => {
+  const { data, journals, handleClick } = props;
+
+  return (
+    <div className="heat-map">
+      <LogoHeader />
+      <h3 className="heat-map__month">
+        <Icon type="calendar" className="heat-map__icon" />
+        {moment().format('MMMM')}
+      </h3>
+      <div className="heat-map__container">
+        <div className="heat-map__body">
+          <CalenderHeatMap
+            startDate={firstDay}
+            endDate={lastDay}
+            values={data}
+            classForValue={value => {
+              if (!value || !value.count) {
+                return 'color-empty';
+              }
+              return `color-scale-${value.count > 4 ? 4 : value.count}`;
+            }}
+            tooltipDataAttrs={toolTipData}
```

Comment:
@yosefalnajjarofficial I think toolTipData is the exported function from the utils file, which means you still have to call that function:
`tooltipDataAttrs={toolTipData()}`

Reply:
The toolTipDataAttrs take a callback function so it works fine like this

Reply:
Ah cool